plugins/in_syslog/syslog_prot.c

```diff
@@ -139,7 +142,10 @@ int syslog_prot_process_udp(char *buf, size_t size, struct flb_syslog *ctx)
         flb_free(out_buf);
     }
     else {
-        flb_warn("[in_syslog] error parsing log message");
+        flb_warn("[in_syslog] error parsing log message "
+                 "on \"%s\" with parser '%s')",
+                 flb_input_name(ctx->i_ins), ctx->parser->name);
+	flb_debug("[in_syslog] unparsed log message: %.*s", size, buf);
```

Comment:
please fix the code style here, the current line is using a tab (\t) instead of spaces .

note: make sure the commit does not contain any tab